docs/UserGuide.md

```diff
@@ -461,6 +461,20 @@ Format: `viewSchedule DATE`
 Example:
 `viewSchedule 2022-02-14`
 
+:exclamation: **Caution**
```

Comment:
Can you wrap your caution block in a `<div>` block? For example,

`<div markdown="span" class="alert alert-warning">`

:exclamation: **Caution:**
`// explanation included here`

`</div>`

This generates a coloured rectangle around the section which make sit more eye catching to the users in the user guide.